dashboard/lib/projects_list.rb

```diff
@@ -18,6 +18,11 @@ module ProjectsList
     dance: ['dance']
   }.freeze
 
+  # Sharing of advanced project types to the public gallery is restricted for
+  # young students unless sharing is explciitly enabled by the student's
+  # teacher for privacy reasons.
+  ADVANCED_PROJECT_TYPES = ['applab', 'gamelab']
```

Comment:
What about Sprite Lab?  I don't think it lets you upload/create any images, but it allows arbitrary text and we don't have any PII filtering in place yet.  Maybe we should keep it locked down for now.

Reply:
Good idea.  I didn't think about Sprite Lab.  I'll add and we can revisit later if needed.